src/app/g3w-ol/projection/projection.js

```diff
@@ -19,7 +7,7 @@ const Projection = function(options={}) {
   ol.proj.Projection.call(this, {
     code: epsg,
     //FOR NOW FORCE TO GET EXTENT
-    extent: false ? extent : degrees ? GENERIC_GRID_EXTENT_DEGREE: CUSTOM_PROJECTIONS_EXTENT[epsg] || GENERIC_GRID_EXTENT,
```

Comment:
```js
// FOR NOW FORCE TO GET EXTENT
```

Please avoid using comments that are valid only "for now" (yes but in what time?), or anything else that isn't exactly quantifiable..

Anyway, it seems that this comment is no longer needed (otherwise update it as you see fit)